### ISSUE TITLE
Voice over does not read that a setting is ignored to sync

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#107748]
Testing #107610

- Focus on a setting that is ignored to sync

🐛 Voice over does not read that setting is ignored to sync

<img width="647" alt="image" src="https://user-images.githubusercontent.com/10746682/94610644-debec800-02a0-11eb-979e-4b32ba844b1e.png">

**Comment by rzhao271** [comment]
One setting to check this issue with is `emmet.extensionsPath`. It is one of those settings that settings sync ignores.

**Comment by sandy081** [comment]
@rzhao271 I still see Voice Over not reading it

<img width="882" alt="image" src="https://user-images.githubusercontent.com/10746682/152179655-011b0e36-fab4-4b3d-977c-ff0050964a18.png">

<img width="774" alt="Screenshot 2022-02-02 at 16 02 23" src="https://user-images.githubusercontent.com/10746682/152179581-b47360a6-3d56-4ae3-b934-647de1d04af9.png">

I am reopening it as verification found. Since it is not a blocker moving this to next milestone.
